src/_pytest/fixtures.py

```diff
@@ -124,7 +130,7 @@ def pytest_sessionstart(session: Session) -> None:
 
 def get_scope_package(
     node: nodes.Item,
-    fixturedef: FixtureDef[object],
+    fixturedef: FixtureDef[Any, object],
```

Comment:
what does Any represent here?

Reply:
It's for filling the new generic parameter for the arguments